apps/web/src/server/applications/case/representations/representation-details/representation-details.utilities.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @typedef {import('./application-representation-details.view-model.js').Representation} Representation
```

Comment:
maybe put this in a type file rather than come at the type from all over ?

Single place for rep jsdoc types.

Reply:
done